chance/p5js/chance-contrast/index.md

```diff
@@ -0,0 +1,9 @@
+layout: p5js
+author: Edmundo Mejía Galindo
+author-url: https://github.com/edgalindo/
+title: Hairy Story
+tags: [P5.js, Law of Causality, Law of Connected Elements, Law of Relativity of Properties, form, grid, interaction, repeat]
+permalink: /chance/Contrast/
```

Comment:
please use the title as permalink name and also as folder name

    /chance/hairy-story/

would be a good permalink and folder name.